### PR TITLE
New tool, pVACbind, to make predictions from a FASTA file

### LINKED ISSUE #299: Create an option that allows user to provide FASTA file as starting input
I would like to be able to provide pVACSeq with a FASTA file as input instead of an annotated VCF.

### PR DESCRIPTION
**Pull request by susannasiebert**
TODO:
- We need a pVACbind logo for the docs

Closes #299

### CODE REVIEW

File: docs/pvacbind.rst [hunk 1]
```diff
@@ -0,0 +1,18 @@
+.. image:: images/pVACbind_logo_trans-bg_sm_v4b.png
+    :align: right
+    :alt: pVACbind logo
+
+pVACbind
+====================================
+
+This component of the pVACtools is used to predict neoantigens for the peptides in a fasta file.
```
Comment (tmooney) [inline docs/pvacbind.rst]: The copyeditor in me notes that `fasta` was capitalized `FASTA` in the index.rst. 
 Searching the repo, it appears that both versions are used several times.

File: docs/pvacbind/filter_commands.rst [hunk 2]
```diff
@@ -0,0 +1,52 @@
+.. image:: ../images/pVACbind_logo_trans-bg_sm_v4b.png
+    :align: right
+    :alt: pVACbund logo
```
Comment (tmooney) [inline docs/pvacbind/filter_commands.rst]:
```suggestion
    :alt: pVACbind logo
```

**Comment by susannasiebert** [comment]
@tmooney I made your suggested changes and fixed a couple other issues.

**Comment by susannasiebert** [comment]
@CodyRamirez has run this command successfully so I will merge this PR. If we find any other bugs with it we can make a new issue.

**Pull request merged**